### DISCUSSION TITLE
RomM on microsoft edge browser of XBOX ONE. Please help me.

### DISCUSSION THREAD

**Discussion by sagaksagak** [gaseous-project/gaseous-server#429] [General Q&A] [answered]
After installing EmulatorJS on my NAS, I enjoyed games through the Xbox One's Edge browser. The Edge browser has a feature to switch the Xbox controller between web browsing mode and game controller mode. This can be done by pressing the ▤ button (three-line button, normally the start button). With EmulatorJS, since the library selection screen is in web browsing mode before launching a game, pressing this screen and switching modes allows the game to run in full screen, and all players (1P and 2P) can use the Xbox One controller normally. Alternatively, you can switch modes at any time during gameplay by pressing the start button.
However, with RomM, after pressing the play button in the game library, switching to controller mode in an empty space of the Edge browser doesn't work in the game. Or if it does work, only 1P functions while 2P doesn't. When entering the game controller settings button located in the bottom left of the game screen, button mapping is set for 1P, but not for 2P. All settings for 2P are blank.
Additionally, pressing the start button on the game screen brings up a small dialog (screenshot/recording/save/load dialog) instead of the controller switching menu, which is very inconvenient.
I would greatly appreciate it if you could pay more attention to controller support on Xbox One.

**Comment by michael-j-green** [comment]
Hi @sagaksagak; this is the Gaseous GitHub... I think you're looking for the Romm one located here: https://github.com/rommapp/romm

**Reply by michael-j-green** [reply]
Ping @gantoine

**Reply by gantoine** [reply]
all the more reason we should just merge our projects already ;)

**Comment by gantoine** [comment, marked as answer]
thanks @michael-j-green, i opened an issue on our end https://github.com/rommapp/romm/issues/1196
you're can lock and delete this discussion if you wish!

**Reply by michael-j-green** [reply]
Thanks! :)
